docs/spec/spec.md

```diff
@@ -59,13 +64,20 @@ Schema: [`LoginSuccessful.schema.json`](../schemas/LoginSuccessful.schema.json)
         "expires_in": 3600
     }
 
-Otherwise an error response is returned with status code 401 if the token was invalid, or 403 if the user is not
- a member of the configured GitHub organization or team.
+Otherwise an error response is returned with status code 401 if the token was invalid, or 403 if the authentication 
+provider is GitHub and the user is not a member of the configured GitHub organization or team.
+
+## GET /
+
+Return a description of all endpoint available over the api. 
```

Comment:
`endpoint` -> `endpoints`

Reply:
Good question re 'general points', I really wasn't sure what was meant by that! I'll have a look and see if I can glean anything...